packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEditForm.tsx

```diff
@@ -46,23 +46,13 @@ export const InboundLineEditForm: FC<InboundLineEditProps> = ({
       </ModalRow>
 
       {item && (
-        <ModalRow>
-          <Grid style={{ display: 'flex', marginTop: 10 }} flex={1}>
-            <ModalLabel label={t('label.code')} />
-            <BasicTextInput disabled sx={{ width: 150 }} value={item.code} />
-          </Grid>
-          <Grid
-            style={{ display: 'flex', marginTop: 10 }}
-            justifyContent="flex-end"
-            flex={1}
-          >
-            <ModalLabel label={t('label.unit')} justifyContent="flex-end" />
-            <BasicTextInput
-              disabled
-              sx={{ width: 150 }}
-              value={item.unitName}
-            />
-          </Grid>
+        <ModalRow margin={3}>
```

Comment:
made it like the outbound version, where it was requested that the code be dropped as a separate field